src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/steam/SteamParallelMultiblockMachine.java

```diff
@@ -44,13 +44,14 @@
 @MethodsReturnNonnullByDefault
 public class SteamParallelMultiblockMachine extends WorkableMultiblockMachine implements IDisplayUIMachine {
 
-    public static final int MAX_PARALLELS = 8;
+    public int MAX_PARALLELS = ConfigHolder.INSTANCE.machines.steamMultiParallelAmount;
 
     // if in millibuckets, this is 0.5, Meaning 2mb of steam -> 1 EU
     private static final double CONVERSION_RATE = 0.5D;
 
```

Comment:
```suggestion
public SteamParallelMultiblockMachine(MBE holder) { this(holder, ConfigHolder.INSTANCE.machines.steamMultiParallelAmount); }
```
Make another constructor that will, by default, use the configured steam parallel amount.
Then, in the registration for the steam oven and grinder, just change it back to what it was before. instead of passing the config value there.